docs/book/src/user/installing.md

```diff
@@ -0,0 +1,119 @@
+# Installation
```

Comment:
This is the quick start and shouldn't be renamed. There are multiple links to it in the wild